### ISSUE TITLE
Automatically copy old civx wallet.dat

### ISSUE THREAD

**Issue by erseco** [exoeconomy/EXOS-Qt#16]
The new EXOS-Qt should copy the old CIVX-qt wallet.dat to the new EXOS-Qt, or provide some assistance to the users:

To do it in macOS:
`cp ~/Library/Application\ Support/CivX/wallet.dat ~/Library/Application\ Support/EXOS`

**Comment by wxmerkt** [comment]
There appears to be a "migrator" tool under releases - but I'd like to now what it does under the hood besides copying the wallet files? Especially since it installs itself.

**Comment by turcol** [comment]
The migrator tool **ONLY** copy the file wallet.dat under the places below. Before to copy the file, the migrator tool backup the wallet.dat file under EXOS directory. 

**Be sure to have both wallets closed before to copy the file.**

```
macOS
cp ~/Library/Application\ Support/CivX/wallet.dat ~/Library/Application\ Support/EXOS
```
```
Windows
cp %appdata%/CivX/wallet.dat %appdata%/EXOS/wallet.dat
```
The videos showing how to migrate the tokens, either manual or using the Migration Tool, are under second review. We expect the new versions to be released this week

**Comment by wxmerkt** [comment]
Thank you @turcol - so the above copy command replaces use of the migrator tool?

**Comment by turcol** [comment]
@wxmerkt , copying the wallet.dat file to the new directory (and backing the wallet.dat under /EXOS directory for safety reasons) replace the migrator tool.

Sorry the final videos were not released yet, which will make clear both options to migrate from CIVX to **EXOS**

Here the first version of the videos. Second version will have sound (voice or music), and the text explaining how to copy the file. Please feel to drop your feedback about the videos:
```
https://drive.google.com/drive/folders/1u3F1jLNxEnPdiHw5YvTHE_RWgMBYOq9l?usp=sharing
```
@wxmerkt you are safe with just copying the file.

**Comment by wxmerkt** [comment]
Thank you @turcol (and videos look great/clear). Given that both use the same `wallet.dat` is the deadline for conversion from one wallet to another (April 30th) less strict than it appears?

**Comment by turcol** [comment]
Good point ! The wallet.dat basically has the same format and contain the private keys (access to your tokens), however the wallet (Qt) new release (2.0.0.1) is preparing for changes relating to the coming releases. The migration is tight to block **500,000** that we expect to hit around the end of April.

You can upgrade your wallet after the deadline without compromising your current tokens. If you are **Staking** it highly recommends upgrading to version 2.0.0.1 before the deadline

**Comment by wxmerkt** [comment]
Thank you for the clarification @turcol :-)
